react-app/src/Finish/index.js

```diff
@@ -4,8 +4,37 @@ import Header from './Header'
 import Image from './Image'
 import Buttons from './Buttons'
 import Lottie from 'lottie-react-native'
+import { useEffect, useState } from 'react'
+import * as SQLite from 'expo-sqlite'
 
 export default function Finish() {
+  const db = SQLite.openDatabase('shelter.db')
+  const [debugMessage, setDebugMessage] = useState('')
+
+  useEffect(() => {
+    const timestamp = new Date().getTime()
+    db.transaction(
+      (tx) => {
+        tx.executeSql(
+          `insert into Record (timestamp, exercise_name) values (?,?);`,
```

Comment:
The table have 3 column but just insert 2 value here, might cause an error.

Reply:
I think this is alright! Thank you for double checking!